### DISCUSSION TITLE
iPad App - “Attempting to save this note has failed. The note cannot be found.”

### DISCUSSION THREAD

**Discussion by zurt** [standardnotes/forum#2828] [General]
**Describe the bug**

After waking my iPad and opening SN, I found that the text I had added earlier was missing from note. I hadn’t used SN on any other device since adding that text. But I do regularly use SN on an iPhone with the app and on a work computer with Safari.

While troubleshooting, I looked at the note history and did not see a record of the last text entry. I did not see the missing text anywhere in the history so I suspected that the app had failed to write the changes to storage.

Then I tried to create a new note as a test. After the new note was created, as soon as I started typing into the text area the following error appeared:

“Attempting to save this note has failed. The note cannot be found.” (Screenshot below)

I had to force quit the app and start over.

**To Reproduce**
1. Enter text in a note
2. Sleep the iPad
3. Wake the iPad and return to SN app
4. Text is missing
5. View the note history and the change log is missing
6. Create a new note
7. Start typing in the text area
8. Error pops up: “Attempting to save this note has failed. The note cannot be found.” (Screenshot below)

**Expected behavior**

* Text should never disappear.
* New notes should be editable.

**Screenshots**

![IMG_0220](https://user-images.githubusercontent.com/301343/178080001-bab71bd1-6084-4047-b2e3-0d6dae912c12.jpeg)

**Smartphone (please complete the following information):**
 - Device: iPad Pro (11-inch) (3rd generation)
 - OS: iOS 15.5
 - App: Standard Notes
 - Version: 3.23.8 (10)

**Additional context**

This has happened several times in the past couple weeks.

This error occurs while I am using the SN app on my iPad. But I do regularly use SN on an iPhone with the app and on a work computer with Safari.

**Comment by onlinemumie** [comment]
Having the same trouble with the same version (3.23.8) on my android devices, Huawei Honor 9 and Lenovo Tab P10, both with android 9.

Have installed version 3.23.3 from APKMirror and deactivated automatic updates on Google Play as a temporary solution. With a little luck, it won't repeat the same error (always handy to have a working apk in downloads when that special update, with your name on it, ruins your day).

Unfortunately, earlier version 3.20.1 don't have file upload support, and some smart tags don't work, so if save error persists in 3.23.3, I guess android workflow will be stifled until next update.
